src/main/java/core/basesyntax/model/Figure.java

```diff
@@ -0,0 +1,7 @@
+package core.basesyntax.model;
+
+public abstract class Figure implements Drawable {
+    public static final int NUMBER_OF_FIGURES = 5;
+    public double area;
+    public Color color;
```

Comment:
let's make these variables private)
remember about encapsulation

Reply:
I aggree about the mistake here, but i can't make them `private` because inherited classes won't be able to have this fields. Is making them `protected` a valid solution in such case?

Reply:
we can make these fields private and create constructor, getters or setters

Reply:
But the area is calculated differently for each figure, and if I have a milti-level inheritance, I cant use parent constructors in order to initialize area field. I consider a workaround with protected setArea and setColor, but is it valid?